particles/Chooser/Chooser.js

```diff
@@ -102,7 +102,7 @@ ${productStyles}
     }
     _render(props, state) {
       return {
-        person: 'Claire',
+        choices: '{{choices.description}}',
```

Comment:
The reason I did it this way was to avoid sending view description provided by one particle to another, because it can expose undesired information (like here, the Chooser just shows a list of products, but it shouldn't know that these come from Claire's wishilist). I want to keep the descriptions on the outer pec side.
I mentioned it in the design doc in the "Using View description in Particles" section.

Let me know what you think.